### PR TITLE
reg-load: add page

### PR DESCRIPTION
**Pull request by pxgamer**
Part of #1908

----

- [x] The page (if new), does not already exist in the repo.

- [x] The page (if new), has been added to the correct platform folder:  
      `common/` if it's common to all platforms, `linux/` if it's Linux-specific, and so on.

- [x] The page has 8 or fewer examples.

- [x] The PR is appropriately titled:  
      `<command name>: add page` for new pages, or `<command name>: <description of changes>` for pages being edited.

- [x] The page follows the [contributing](https://github.com/tldr-pages/tldr/blob/master/CONTRIBUTING.md) guidelines.

### CODE REVIEW

File: pages/windows/reg-load.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+# reg load
+
+> Load saved sub keys into a different sub key in the registry.
+> This is intended for troubleshooting and temporary keys.
+
+- Load a specified key from a backup file:
+
+`reg load {{key_name}} {{path/to/file}}`
```
Comment (sbrl) [inline pages/windows/reg-load.md]: So is this the key name to load it into, or the key name to load selectively from the file?

Comment (pxgamer) [reply pages/windows/reg-load.md]: The key name to load it into.

Comment (sbrl) [reply pages/windows/reg-load.md]: Right. Perhaps `Load a backup file into the specified key` or something might aid clarity here?

Comment (pxgamer) [reply pages/windows/reg-load.md]: Yes, that makes quite a bit more sense.

**Comment by sbrl** [comment]
Thanks, @pxgamer :D

**Pull request merged**